src/components/filterPatterns/modal/OpenFilterButton.tsx

```diff
@@ -21,13 +24,11 @@ type Props = Pick<
     variant?: Variant;
     isDisabled?: boolean;
     paddingX?: PaddingX;
+    showResetButton?: boolean;
```

Comment:
correct me if I am wrong but we always want the reset button, when `onResetFilter` is defined. Is this prop really necessary?